build-tools/scripts/RunNUnitTests.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="RunTests" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
+    <_RuntimeVersion Condition=" '$(OS)' != 'Windows_NT' ">--runtime=v4.0.0</_RuntimeVersion>
```

Comment:
Do we need `--runtime`?

Reply:
Probably not, I was mainly trying to keep everything exactly as it was.

I think the tests still pass fine without it.